.gitlab/ci/build_quartz.yml

```diff
@@ -4,62 +4,79 @@
   tags:
     - shell
     - quartz
-  except:
-    refs:
-      - /_qnone/
-    variables:
-      - $SERAC_CI_QUARTZ == "OFF"
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /_qnone/ || $ON_QUARTZ == "OFF"' #run except if ...
+      when: never
+    - if: '$CI_JOB_NAME =~ /release_resources/'
+      when: always
+    - when: on_success
 
 ####
 # In pre-build phase, allocate a node for builds
 allocate_resources_build_quartz:
   variables:
     GIT_STRATEGY: none
-  extends: .on_quartz
+  extends: [.on_quartz, .pr_workflow]
   stage: allocate_resources
   script:
-    - salloc -N 1 -c 36 -p pdebug -t 30 --no-shell --job-name=${PROJECT_ALLOC_NAME}
+    - salloc ${ACCOUNT} -N 1 -c 36 -t 30 --no-shell --job-name=${PROJECT_ALLOC_NAME}
```

Comment:
This is a more general question I guess.  If this triggering pipelines from upstream projects becomes more common, would it be up to the downstream project to use their account? Is that even something we need to worry about? @jamiebramwell

Reply:
I never thought about it, but the way it is right now, the upstream project could in fact change the ACCOUNT value to use its own...
Here I am using radiuss account after approval of Rob that this is a valid use of it.

Reply:
I can see it both ways.. maybe we shouldn't make a fuss about whose account is used.

Reply:
Not yet... service users should become the norm, once it’s available. We shall re-discuss that then.